### PR TITLE
Fix: Post layout - Conflicting with NexGen.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello - https://trello.com/c/dAWQ0lv9/270-fix-post-layout-nextgen-conflicting-with-post-excerpt

### Screenshots

Issue - https://share.getcloudapp.com/YEuRpnwZ

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install NextGen Gallery plugin.
- Drag and drop any post layout block and update the page it shows the JSON error.
- Switch to `post-layout-conflicting-with-NexGen` and update the page. It updates successfully.
- Add an excerpt to the post from meta settings and then check the post layout at the front end. (It should display an excerpt in this case)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: dist/blocks/post/class-uagb-post.php [hunk 1]
```diff
@@ -1432,7 +1432,7 @@ public function render_excerpt( $attributes ) {
 			if ( 'full_post' === $attributes['displayPostContentRadio'] ) {
 				$excerpt = get_the_content();
 			} else {
-				$excerpt = wp_trim_words( get_the_excerpt(), $length );
+				$excerpt = wp_trim_words( get_the_content(), $length );
```
Comment (RajkiranBagal) [inline dist/blocks/post/class-uagb-post.php:1435]: @AbhijitNage123 Why are we saving whole page content instead of the excerpt?

Comment (AbhijitNage123) [reply dist/blocks/post/class-uagb-post.php:1435]: @RajkiranBagal get_the_excerpt() causing the issue while rendering the excerpt. That's why I use the get_the_content().

Comment (RajkiranBagal) [reply dist/blocks/post/class-uagb-post.php:1435]: Yes, But get_the_content() gives full content of a Page. So The option of showing excerpt will not work.

Comment (AbhijitNage123) [reply dist/blocks/post/class-uagb-post.php]: Resolved

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**